crates/python/qcs_sdk/qpu/translation.pyi

```diff
@@ -154,12 +164,17 @@ class TranslationOptions:
         """
         Use the v2 backend for translation, available on QCS since 2023.
         """
-    def use_q_ctrl(self, q_ctrl: QCtrl = ..., /) -> None:
+    def use_q_ctrl(self, q_ctrl: QCtrl = None, /) -> None:
```

Comment:
I couldn't think of any reason not to make it explicit that the default value is `None` both here and for Riverlane.